sdk/synapse/Azure.Analytics.Synapse.AccessControl/CHANGELOG.md

```diff
@@ -1,4 +1,6 @@
 # Release History
 
-## 1.0.0-preview.1 (Unreleased)
+## 1.0.0-preview.2 (Unreleased)
+
```

Comment:
@pakrym @tg-msft could you please suggest on the change log?

Reply:
Per https://azure.github.io/azure-sdk/policies_releasenotes.html#what-if-my-library-is-changed-but-doesnt-have-any-developer-impacting-changes, we generally suggest:

> This release contains bug fixes to improve quality.